webpack/assets/javascripts/react_app/redux/reducers/hosts/powerStatus/powerStatus.test.js

```diff
@@ -11,30 +11,39 @@ import {
 } from './powerStatus.fixtures';
 
 import reducer from './index';
+import { testReducerEqualityWithFixture } from '../../../../common/testHelpers';
 
 describe('powerStatus reducer', () => {
-  it('should return the initial state', () => {
-    expect(reducer(undefined, {})).toEqual(initialState);
-  });
-
-  it('should handle HOST_POWER_STATUS_REQUEST', () => {
-    expect(reducer(initialState, {
-      type: types.HOST_POWER_STATUS_REQUEST,
-      payload: request,
-    })).toEqual(stateBeforeResponse);
-  });
-
-  it('should handle HOST_POWER_STATUS_SUCCESS', () => {
-    expect(reducer(stateBeforeResponse, {
-      type: types.HOST_POWER_STATUS_SUCCESS,
-      payload: response,
-    })).toEqual(stateAfterSuccess);
-  });
-
-  it('should handle HOST_POWER_STATUS_FAILURE', () => {
-    expect(reducer(stateBeforeResponse, {
-      type: types.HOST_POWER_STATUS_FAILURE,
-      payload: { error, id: request.id },
-    })).toEqual(stateAfterFailure);
-  });
+  const fixtures = {
+    'should return the initial state': {
+      prev: undefined,
+      action: {},
+      next: initialState,
```

Comment:
I personally prefer snapshots rather then supply a preconfigured results object.

Reply:
I will see what I can do about that.